.openpublishing.redirection.json

```diff
@@ -6353,1104 +6353,1104 @@
         },
         {
             "source_path": "docs/framework/wpf/advanced/activate-function-wpf-unmanaged-api-reference.md",
-            "redirect_url": "/dotnet/desktop/framework/wpf/advanced/activate-function-wpf-unmanaged-api-reference?view=netdesktop-5.0"
+            "redirect_url": "/dotnet/desktop/wpf/advanced/activate-function-wpf-unmanaged-api-reference?view=netdesktop-5.0"
```

Comment:
also, do you really need to set the view here? Can't you let the default view render?